docs/getting-started.asciidoc

```diff
@@ -76,9 +76,9 @@ The schema for the logs data set has dozens of different fields, but the notable
     "@timestamp": "date"
 }
 
-Before we load the Shakespeare data set, we need to set up a {ref}/mapping.html[_mapping_] for the fields. Mapping 
-divides the documents in the index into logical groups and specifies a field's characteristics, such as the field's
-searchability or whether or not it's _tokenized_, or broken up into separate words.
+Before we load the Shakespeare and logs data sets, we need to set up {ref}/mapping.html[_mappings_] for the fields. 
```

Comment:
There's an extra `/` in this "mappings" url.  It doesn't seem to break anything, but it's probably worth cleaning up at some point:
![screen shot 2016-04-04 at 4 15 44 pm](https://cloud.githubusercontent.com/assets/53814/14261845/953e2978-fa80-11e5-854c-66783f523bbe.png)